assets/prototype/domain/eventEditor/context/DateTimeProvider/index.tsx

```diff
@@ -9,8 +9,7 @@ import {
 	EditorState,
 } from '../../../../application/ui/components/layout/editor-modal/useEditorModalState/types';
 
-export interface DateTimeContextProps extends ContextProps {}
-export const DateTimeContext = createContext({} as DateTimeContextProps);
+export const DateTimeContext = createContext({} as ContextProps);
```

Comment:
Fix for the:
![14_jan_2](https://user-images.githubusercontent.com/1477453/72341641-65138980-36d3-11ea-87b5-4fb5a9ef99e0.png)